automated/linux/usb/test-usb-3.yaml

```diff
@@ -0,0 +1,19 @@
+metadata:
+    name: Test usb-3
+    format: "Lava-Test Test Definition 1.0"
+    description: "Read function."
```

Comment:
Missing a little bit description, something like "Test read speed from USB flash disk."
Or is this test doing something different?

Reply:
Each description has been updated.